tests/type-tests/test/orchestration.test-d.ts

```diff
@@ -244,6 +246,26 @@ expectType<Promise<OrchestrationResponse>>(
 expect<ChatModel>('custom-model');
 expect<ChatModel>('gemini-1.0-pro');
 
+/**
+ * FilteringUtil.
+ */
+
+expectType<AzureContentSafetyFilterConfig>(
+  ContentFilters.buildAzureContentSafety({
+    Hate: 'ALLOW_ALL',
+    SelfHarm: 'ALLOW_SAFE_LOW',
+    Sexual: 'ALLOW_SAFE_LOW_MEDIUM',
+    Violence: 'ALLOW_SAFE'
+  })
+);
+
+expectError<AzureContentSafetyFilterConfig>(
+  ContentFilters.buildAzureContentSafety({
+    Hate: 2,
+    SelfHarm: 4
+  })
+);
```

Comment:
@jjtang1985 @deekshas8 This is different behaviour from what we discussed yesterday. But since we are only introducing constants in the new convenience function, I thought we can also just stick to string constants, instead of allowing numeric values as well.

Please tell me if you feel otherwise.